MetaMask/metamask-mobile
# feat(MMPD-546): update portfolio icon

**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

Due to a redesign, we are changing the link to portfolio from a simple icon to instead be a secondary button. 

**Screenshots/Recordings**

<img width="385" alt="Screenshot 2023-07-06 at 12 52 45 PM" src="https://github.com/MetaMask/metamask-mobile/assets/22625602/1e2af7ff-80b3-4d1c-bb5b-11332a33d1fd">

<img width="375" alt="Screenshot 2023-07-12 at 4 47 40 PM" src="https://github.com/MetaMask/metamask-mobile/assets/22625602/cf12a69e-87cc-45e3-8f13-654b2e0154b1">


How to test:

1. Go to the main view
2. click the portfolio button and make sure the portfolio link works like before

**Issue**

Progresses #???

**Checklist**
